### DISCUSSION TITLE
using-ssh-and-localhost-run-to-test-github-webhooks-locally/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#26] [Announcements]
# using-ssh-and-localhost-run-to-test-github-webhooks-locally/

In this post I describe how to test GitHub webhooks locally using SSH and localhost.run, a free service for HTTP tunnelling.

https://andrewlock.net/using-ssh-and-localhost-run-to-test-github-webhooks-locally/

**Comment by LeoPaulD** [comment]
Thanks for this tutoriel

I follow this but when I make delivery i've this error in my company -> 
```
 We couldn’t deliver this payload: **EOF** 
```
Certainly because interactions with the outside world are cut off?


So I've tried it at home, but I've these messages ->
```   Content-Length: 0
      Serveur: AirTunes/620.8.2
```

or

```
     A tunnel is up but I couldn't connect to a local app, check your SSH output for clues. 
```
I don't know what's wrong 


Have a nice day

Léo-Paul
